app/components/AppBreadCrumb.tsx

```diff
@@ -29,7 +29,7 @@ const useCrumbs = (path: string): Crumb[] =>
     });
     if (parts.length > 1) {
       crumbs.push({
-        label: parts[1].split("?")[0],
+        label: decodeURI(parts[1].split("?")[0]),
```

Comment:
Do you think this `auth0|<long string of hex>` could be replaced with the user's name? I agree that this is highly accurate, but I think it might be clearer if we breadcrumb with something human readable.

Reply:
I do—i'll open a follow-up issue to refine that breadcrumb component. At the moment it's just rendering data from the `pathName` and we'll need to get fancy in order to customize this per page route 👍